workspace.code-workspace

```diff
@@ -133,8 +133,10 @@
     },
     "vue.complete.casing.props": "kebab",
     "vue.inlayHints.missingProps": true,
-    "circleci.persistedProjectSelection": ["gh/specklesystems/speckle-server"],
-    "eslint.experimental.useFlatConfig": true
+    "circleci.persistedProjectSelection": ["gh/specklesystems/speckle-server"]
+    // "vitest.nodeEnv": {
```

Comment:
nit: comment from testing (?)

Reply:
intentionally left it there for posterity